browser.js

```diff
@@ -293,6 +306,7 @@ Socket.prototype.connect = function(options, cb) {
 		hostname: getProxy().hostname,
 		port: getProxy().port,
 		path: getProxy().path + '/connect',
+		protocol: getProxy().requestProtocol,
```

Comment:
allows to use local proxy started with HTTP: on the production website (why? because the dev website build can be broken: assets etc).